docs/whatsnew_1.5.7.html

```diff
@@ -40,22 +39,22 @@
     </section> <!-- End pageHeader //-->
     
     <section id="zenTeamMessage">
-        <h1>Welcome to Zen Cart&reg;</h1>    
```

Comment:
I would suggest removing the remaining &trade; symbols; doing it once is enough.  It's way overkill to do it every time.

Reply:
ok